packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -113,6 +139,12 @@ export class WaferMap extends FoundationElement {
 
     public override connectedCallback(): void {
         super.connectedCallback();
+        const canvasContext = this.canvas.getContext('2d', {
+            willReadFrequently: true
+        });
+        if (canvasContext != null) {
```

Comment:
Do you know what circumstances `canvasContext` will be `null`? When that happens, what will cause `this.canvasContext` to be set?

I'm wondering if we can remove this `if` if it will never be hit or if it's harmless to set `this.canvasContext` to `null` since it will always be populated later.